client/verta/verta/operations/monitoring/profilers.py

```diff
@@ -190,6 +287,7 @@ def __init__(self, conn, conf, client):
         self._conf = conf
         self._client = client
 
+    # TODO: Hide this as an internal method.
```

Comment:
Are `list()` and `get()` meant to be internal? I thought they were intended to be user-facing 😮

Reply:
No I think that comment may have moved around when it was supposed to earmark the `create` method as internal, which only creates a profiler reference.